build.bash

```diff
@@ -264,15 +270,14 @@ if [ "$hw_platform" == "pi-raspios32" ] || [ "$hw_platform" == "pi-raspios64beta
   cp $sourcefolder/openhabian-installer.service $buildfolder/root/etc/systemd/system/
   ln -s $buildfolder/root/etc/systemd/system/openhabian-installer.service $buildfolder/root/etc/systemd/system/multi-user.target.wants/openhabian-installer.service
 
-  echo_process "Setting default runlevel multiuser.target... "
-  ( # open a subshell, returns to current dir automatically
-    cd $buildfolder/root/etc/systemd/system/ || exit 1
-    rm -rf default.target
-    ln -s ../../../lib/systemd/system/multi-user.target default.target
-    # disable autologin
-    rm -f $buildfolder/root/etc/systemd/system/getty@tty1.service.d/autologin.conf
-  )
+  echo_process "Setting default runlevel multiuser.target and disabling autologin... "
+  rm -rf $buildfolder/root/etc/systemd/system/default.target
```

Comment:
using the separate shell was intended to protect building system from errors (empty variable $buildfolder), so please leave it in... maybe it looks a bit paranoid, but still...

Reply:
Ok, will do.